truss/cli/train.py

```diff
@@ -93,10 +100,88 @@ def display_training_jobs(console: Console, jobs, title="Training Job Details"):
     for job in jobs:
         table.add_row(
             job["training_project_id"],
+            job["training_project"]["name"],
             job["id"],
             job["current_status"],
             job["instance_type"]["name"],
             job["created_at"],
             job["updated_at"],
         )
     console.print(table)
+
+
+def view_training_details(
+    console: Console,
+    remote_provider: BasetenRemote,
+    project_id: Optional[str],
+    job_id: Optional[str],
+):
+    """
+    view_training_details shows a list of jobs that meet the provided project_id and job_id filters.
+
+     If no filters are provided, the command will show a list of all training projects and a list of active jobs.
+    """
+    if job_id or project_id:
+        jobs_response = remote_provider.api.search_training_jobs(
+            project_id=project_id,
+            job_id=job_id,
+            order_by=[{"field": "created_at", "order": "asc"}],
+        )
+        if len(jobs_response) == 0:
+            raise click.UsageError("No training jobs found")
+        display_training_jobs(console, jobs_response)
+    else:
+        projects = remote_provider.api.list_training_projects()
+        table = rich.table.Table(
```

Comment:
super nit: Could we move this to a `display_training_projects` helper to keep the intent of this function easier to follow? Through L159 or so